src/renderer/viz/expressions/Animation.js

```diff
@@ -207,7 +208,7 @@ export class Animation extends BaseExpression {
      *   document.getElementById('timestamp').innerHTML = currTime;
      * });
      *
-     * @memberof carto.expressions.Animation
+     * @memberof s.Animation
```

Comment:
memberof `s` works fine?

Reply:
As we saw, this is intended to place these expressions in the sidebar, I am thinking a better way to do so.